Does not compile on ARM Ubuntu

```
In file included from /home/piotr/Work/GitHub/cute_framework/src/cute_audio.cpp:25:
/home/piotr/Work/GitHub/cute_framework/libraries/cute/cute_sound.h:728:18: fatal error: xmmintrin.h: No such file or directory
  728 |         #include <xmmintrin.h>
      |                  ^~~~~~~~~~~~~
compilation terminated.
gmake[2]: *** [CMakeFiles/cute.dir/build.make:104: CMakeFiles/cute.dir/src/cute_audio.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:684: CMakeFiles/cute.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

Possible fix: https://github.com/darktable-org/darktable/commit/d10c290fb3b47bbc8fd90d6d1d7935b900c1caf6

--- a/include/cute_defines.h
+++ b/include/cute_defines.h
@@ -22,6 +22,9 @@
 #	define CF_WINDOWS 1
 #elif defined(__linux__) || defined(__unix__) && !defined(__APPLE__) && !defined(__EMSCRIPTEN__)
 #	define CF_LINUX 1
+#       if !defined(__SSE__)
+#               define CUTE_SOUND_SCALAR_MODE
+#       endif
 #elif defined(__APPLE__)
 #	define CF_APPLE 1
 #	include <TargetConditionals.h>

